src/handlers/handleAudio.ts

```diff
@@ -132,8 +138,8 @@ async function sendTranscription(ctx: Context, url: string, fileId: string) {
             parse_mode: 'Markdown',
           }
         )
-      } catch (error) {
-        report(error, { ctx, location: 'updateMessagewithError' })
+      } catch (err) {
```

Comment:
Don't shorten the `error` variable name :) there is literally no need for that

Reply:
Yes, indeed no need, but just to ignore a warning that they were already defined in upper scope.

Reply:
I see. Can we call them `sendTranscriptionError` in the upper scope then? To avoid confusion

Reply:
for sure! I'll change that 🎯